scripts/generate_syncset.py

```diff
@@ -27,33 +55,72 @@ def get_all_yaml_obj(file_paths):
         objects = get_yaml_all(file)
         for obj in objects:
             yaml_objs.append(obj)
+    yaml_objs = sorted(yaml_objs)
     return yaml_objs
 
+def process_yamls(name, directory, obj):
```

Comment:
nit: "process" is a bit vague here to someone unfamiliar (me). Might suggest adding a docstring with more details of the intention.